packages/datetime/test/dateRangeInputTests.tsx

```diff
@@ -67,100 +67,154 @@ describe("<DateRangeInput>", () => {
         expect(component.find(InputGroup).length).to.equal(2);
     });
 
-    it("startInputProps.inputRef receives reference to HTML input element", () => {
-        const inputRef = sinon.spy();
-        // full DOM rendering here so the ref handler is invoked
-        mount(<DateRangeInput startInputProps={{ inputRef }} />);
-        expect(inputRef.calledOnce).to.be.true;
-        expect(inputRef.firstCall.args[0]).to.be.an.instanceOf(HTMLInputElement);
+    it("shows empty fields when no date range is selected", () => {
+        const { root } = wrap(<DateRangeInput />);
+        assertInputTextsEqual(root, "", "");
     });
 
-    it("startInputProps.onChange receives change events in the start field", () => {
-        const onChange = sinon.spy();
-        const component = mount(<DateRangeInput startInputProps={{ onChange }} />);
-        changeStartInputText(component, "text");
-        expect(onChange.calledOnce).to.be.true;
-        expect((onChange.firstCall.args[0].target as HTMLInputElement).value).to.equal("text");
-    });
+    describe("startInputProps and endInputProps", () => {
 
-    it("endInputProps.inputRef receives reference to HTML input element", () => {
-        const inputRef = sinon.spy();
-        // full DOM rendering here so the ref handler is invoked
-        mount(<DateRangeInput endInputProps={{ inputRef }} />);
-        expect(inputRef.calledOnce).to.be.true;
-        expect(inputRef.firstCall.args[0]).to.be.an.instanceOf(HTMLInputElement);
-    });
+        type MountFunction = (inputGroupProps: HTMLInputProps & IInputGroupProps) => any;
+        type InputGetterFunction = (root: WrappedComponentRoot) => WrappedComponentInput;
 
-    it("endInputProps.onChange receives change events in the end field", () => {
-        const onChange = sinon.spy();
-        const component = mount(<DateRangeInput endInputProps={{ onChange }} />);
-        changeEndInputText(component, "text");
-        expect(onChange.calledOnce).to.be.true;
-        expect((onChange.firstCall.args[0].target as HTMLInputElement).value).to.equal("text");
-    });
+        describe("startInputProps", () => {
+            const _mount = (inputGroupProps: HTMLInputProps & IInputGroupProps) => {
+                return mount(<DateRangeInput startInputProps={inputGroupProps} />);
+            };
+            runTestSuite(_mount, getStartInput);
```

Comment:
swap the param order and you can write this inline and only declare the types once, in the `runTestSuite` signature.

```ts
runTestSuite(getStartInput, (props) => { ... });
```